package.json

```diff
@@ -67,6 +70,7 @@
     "mocha": "^6.2.0",
     "null-loader": "^4.0.1",
     "threads-plugin": "^1.3.1",
+    "typescript": "^4.3.5",
```

Comment:
My last commit adds a dev dependency of `typescript` along with a `test-types` NPM script and a `test.ts` file in the tests folder.  Just run the script and ensure no compile errors and that there is output and no assertion errors.  It's the lightest option I could think of to make it easy to test the types from within the repo (otherwise I have to npm build/link externally) but not let typescript get in the way or bloat the library.

If this is reasonable I can add to the geoblaze PR as well.  If it's not reasonable I can remove it.